vespa-hadoop/src/main/java/com/yahoo/vespa/hadoop/pig/VespaDocumentOperation.java

```diff
@@ -68,10 +68,18 @@ public static boolean valid(String text) {
     private static final String SIMPLE_ARRAY_FIELDS = "simple-array-fields";
     private static final String SIMPLE_OBJECT_FIELDS = "simple-object-fields";
     private static final String CREATE_TENSOR_FIELDS = "create-tensor-fields";
+    private static final String REMOVE_TENSOR_FIELDS = "remove-tensor-fields";
+    private static final String ADD_TENSOR_FIELDS = "add-tensor-fields";
+    private static final String REMOVE_BAG_AS_MAP_FIELDS = "remove-bag-as-map-fields";
+    private static final String ADD_BAG_AS_MAP_FIELDS = "add-bag-as-map-fields";
```

Comment:
From pig user perspective, `remove/add-bag-as-map-fields` may be confused.
Users need to know that there is a functionality to transform bag as map.
Then he/she may know that these two feature is to add/remove a data in map.
Most of the pig users don't need to know the detail of this Java UDF.
It's better to rename them so that pig users would understand what are these for easily.